test/system/700-play.bats

```diff
@@ -394,6 +394,39 @@ status: {}
     run_podman rm -f -t0 myyaml
 }
 
+@test "podman play with init container" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+
+testUserYaml="
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+spec:
+  containers:
+  - command:
+    - ls
+    - /dev/shm/test1
+    image: $IMAGE
+    name: testCtr
+  initContainers:
+  - command:
+    - touch
+    - /dev/shm/test1
+    image: $IMAGE
+    name: initCtr
+"
+    echo "$testUserYaml" > $PODMAN_TMPDIR/test.yaml
+    run_podman kube play $PODMAN_TMPDIR/test.yaml
+    assert "$output" !~ "ERRO" "init containers should not generate logrus.Error"
+    run_podman inspect --format "{{.State.ExitCode}}" pod-testCtr2
+    is "$output" "0" "init container should have created /dev/shm/test1"
+
+    run_podman kube down $PODMAN_TMPDIR/test.yaml
+    run_podman rmi --all
```

Comment:
Please don't do `rmi --all`, that forces a re-fetch of `$IMAGE` which of course means new possibilities for network flakes.